games-fps/alephone-durandal/alephone-durandal-20220115.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit desktop
+
+DESCRIPTION="Aleph One - Marathon 2 (Durandal)"
+HOMEPAGE="http://marathon.sourceforge.net/"
+SRC_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-${PV}/Marathon2-${PV}-Data.zip"
+
+LICENSE="bungie-marathon"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+RESTRICT="mirror"
+
+RDEPEND="games-fps/alephone"
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}/Marathon 2"
+
+MY_NAME='durandal'
+MY_DIR="/usr/share/alephone-$MY_NAME"
```

Comment:
Per https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101 we use ${MY_NAME} style

Reply:
That'd be a good one for pkgcheck / the QA bot to warn about! (Not that I shouldn't have known better here)